### ISSUE TITLE
Javascript error in Firefox

### ISSUE THREAD

**Issue by payten** [steveswinsburg/gradebookNG#13]
Keyboard navigation etc doesn't work in Firefox! (gradebook-grades.js errors)

```
 [13/03/2015 6:47:19 am] Steve Swinsburg: TypeError: this.$cell.find(...)[0].innerText is undefined
 [13/03/2015 6:47:27 am] Steve Swinsburg:     columnKey = this.$cell.find(".gb-title, span:first")[0].innerText.trim();
 [13/03/2015 6:47:29 am] Steve Swinsburg: line 551
```

**Comment by payten** [comment]
Fixed in #14.

**Comment by steveswinsburg** [comment]
This issue was moved to sakaiproject/sakai#1332
